lib/response-transforms.js

```diff
@@ -32,4 +32,8 @@ module.exports = {
       });
     }
   },
+  jsonify: res => {
```

Comment:
Factored it out into its own function so that the promise chains would look cleaner, since we do it more than once and there's every likelihood to expect to have to do it more as things are added if they don't return json